homework/week-0/AllisonBojarski/Cards(HWproblem#2)/src/nyc/c4q/alizinha/PlayingCards.java

```diff
@@ -0,0 +1,30 @@
+package nyc.c4q.alizinha;
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * Created by c4q-Allison on 5/2/15.
+ */
+public class PlayingCards extends Game
+{
+    ArrayList<Card> deck;
+
+    public PlayingCards() {
+
```

Comment:
If you add all of the numbers (A, 2, 3....., Q, K) and the suits it automatically works out to 52 unique cards with your nested loop logic here. I believe one of the specs for this assignment was to generate 52 pieces(Cards).

Aside from that, nice nested loop!